/mnt/c/Users/sadam/Desktop/Research/PHO_Project/ERA5_PredRNN/data/PDE_47c1f778-4bf1-46bc-a27a-e60a268ba2fb/data.npz
train_data_files nums: 1
mnist test iterator, Loading data from /mnt/c/Users/sadam/Desktop/Research/PHO_Project/ERA5_PredRNN/data/PDE_cb9898ef-3e5a-4271-a856-eb73704f1986/data.npz
NaN value num: 0
mnist train iterator, Loading data from /mnt/c/Users/sadam/Desktop/Research/PHO_Project/ERA5_PredRNN/data/PDE_47c1f778-4bf1-46bc-a27a-e60a268ba2fb/data.npz
NaN value num: 0
Traceback (most recent call last):
  File "../predrnn-pytorch/run2.py", line 357, in <module>
    train_wrapper(model)
  File "../predrnn-pytorch/run2.py", line 278, in train_wrapper
    ims = train_input_handle.get_batch()
  File "/mnt/c/Users/sadam/Desktop/Research/PHO_Project/ERA5_PredRNN/predrnn-pytorch/core/data_provider/mnist_new.py", line 133, in get_batch
    input_seq = self.input_batch()
  File "/mnt/c/Users/sadam/Desktop/Research/PHO_Project/ERA5_PredRNN/predrnn-pytorch/core/data_provider/mnist_new.py", line 128, in input_batch
    input_batch[i, :self.current_input_length, :, :, :] = data_slice
ValueError: could not broadcast input array from shape (4,1,64,64) into shape (48,1,64,64)